package.json

```diff
@@ -50,7 +50,7 @@
     "test": "yarn run hardhat test",
     "test:ci": "./scripts/run-tests.ci.sh",
     "deploy:devchain": "yarn run hardhat deploy --network devchain && ts-node --files ./scripts/mineBlocks.ts",
-    "devchain": "yarn run celo-devchain --port 7545 --core v7",
+    "devchain": "yarn run celo-devchain --port 7545 --file ./devChain/devchain.tar.gz",
```

Comment:
did you mean to delete that `core` flag?

Reply:
yep, unfortunately if we want to run tests we need newest tar since the changes are not yet included in core contracts